finish/src/main/java/io/openliberty/guides/bulkhead/global/eBank/microservices/BankService.java

```diff
@@ -8,25 +8,25 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package global.eBank.microservices;
+package io.openliberty.guides.bulkhead.global.eBank.microservices;
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
 
 import org.eclipse.microprofile.faulttolerance.exceptions.BulkheadException;
 import org.eclipse.microprofile.faulttolerance.Asynchronous;
 import org.eclipse.microprofile.faulttolerance.Bulkhead;
+import org.eclipse.microprofile.faulttolerance.Fallback;
 
-//import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.Future;
 import java.util.concurrent.CompletableFuture;
 
-import global.eBank.microservices.Service;
+import io.openliberty.guides.bulkhead.global.eBank.microservices.Service;
```

Comment:
you don't need to import Service as it has the same package and located in the same directory.